apis/nucleus/src/components/glue.jsx

```diff
@@ -5,11 +5,13 @@ import Cell from './Cell';
 export default function glue({ halo, element, model, initialSnOptions, initialSnPlugins, onMount, initialError }) {
   const { root } = halo;
   const cellRef = React.createRef();
+  const currentId = String(Date.now()); // TODO: different ID generator?
```

Comment:
yes use a different ID generator, Date.now() will not create unique values if called to fast